.pre-commit-config.yaml

```diff
@@ -59,7 +59,8 @@ repos:
         exclude: |
           (?x)^(
             platform/android/java/editor/src/main/java/com/android/.*|
-            platform/android/java/lib/src/com/.*
+            platform/android/java/lib/src/com/.*|
+            tests/.*\.txt
```

Comment:
Looks unintended.

Reply:
It was processing sam's file tree dump, but since that's test data anyways I'll apply it and back out this change